bower.json

```diff
@@ -2,6 +2,8 @@
   "name": "ember-appboy",
   "dependencies": {
     "ember": "~2.8.0",
-    "ember-cli-shims": "0.1.1"
+    "ember-cli-shims": "0.1.1",
+    "hammer.js": "^2.0.8",
```

Comment:
this is from ember-paper, which I'm using for the demo site. fear not, it won't be included in the consumer's application.